developer/src/kmcmplib/src/Compiler.cpp

```diff
@@ -3362,6 +3363,18 @@ int atoiW(PKMX_WCHAR p)
   return i;
 }
 
+bool isuiw(PKMX_WCHAR p) {
+  if (!p || !*p)
+    return false;
+  PKMX_STR q = wstrtostr(p);
```

Comment:
This is leaking memory because `q` is not deleted

Reply:
Good catch ... I am spending too much time with more modern languages. Corrected.